src/test/playground.test.ts

```diff
@@ -119,4 +125,21 @@ describe.skipIf(!playgroundTrue)("playground", () => {
     expect(status).toBe("success")
     expect(balanceAfter - balanceBefore).toBe(1n)
   })
+
+  test("should send a user operation using nexusClient.sendUserOperation", async () => {
+    const balanceBefore = await publicClient.getBalance({
+      address: recipientAddress
+    })
+    const userOpHash = await nexusClient.sendUserOperation({
```

Comment:
Want to compare this userOpHash to nexusClient.account.getUserOpHash() ?

Reply:
Can you explain why this: 
![image](https://github.com/user-attachments/assets/1af12708-5ed4-4a24-8d06-a42ecadb7086)
Is preferable over this
![image](https://github.com/user-attachments/assets/313d44a9-11a4-4222-915b-45327f33aa98)

I guess the entrypoint always exists right (our factory may not yet be deployed)

Reply:
One had to be removed, I chose to remove the method using the k1 validator factory since that might change while the entrypoint will never change.